contracts/axelar-gateway/src/event.rs

```diff
@@ -30,7 +30,19 @@ pub(crate) fn execute_contract_call(env: &Env, message_id: String) {
     env.events().publish(topics, ());
 }
 
-pub(crate) fn rotate_signers(env: &Env, signers: WeightedSigners) {
-    let topics = (symbol_short!("rotated"),);
-    env.events().publish(topics, (signers,));
+pub(crate) fn rotate_signers_set(
```

Comment:
```suggestion
pub(crate) fn rotate_signers(
```
keep the same name since it matches the entrypoint?

Reply:
updated.